src/paperless/settings.py

```diff
@@ -248,6 +251,9 @@ def deprecated(before, after):
         deprecated("PAPERLESS_CONVERT", "PAPERLESS_CONVERT_BINARY")
         CONVERT_BINARY = os.getenv("PAPERLESS_CONVERT", CONVERT_BINARY)
 
+if not UNPAPER_BINARY:
```

Comment:
Is this necessary?  Given that the name is unchanged, why not set [line 193](https://github.com/danielquinn/paperless/pull/74/files#diff-21758e26d686720f35a56d797f195c76R193) to this instead?

```python
UNPAPER_BINARY = os.getenv("PAPERLESS_UNPAPER_BINARY", "unpaper")
```